save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -717,7 +724,7 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
         }
     }
 
-    private fun deleteOrganizationBuilder() {
```

Comment:
Why this method removed?

Reply:
I wanted to make this method an extension function